yarn.lock

```diff
@@ -6022,6 +6051,8 @@ execa@^3.0.0:
     signal-exit "^3.0.2"
     strip-final-newline "^2.0.0"
 
+=======
+>>>>>>> f3699f8... config: update TS and TS test matrix
```

Comment:
@chriskrycho looks like merge conflicts to me 😉

Reply:
Ahhhh, good catch, thank you.